pkg/vendir/fetch/helmchart/http_source.go

```diff
@@ -42,13 +43,24 @@ func (t *HTTPSource) Fetch(dstPath string, tempArea ctlfetch.TempArea) error {
 	return t.fetch(helmHomeDir, dstPath)
 }
 
+func setEnv(helmHomeDir string) []string {
+	re := regexp.MustCompile(`(?i)^(https?|no)_proxy=.*`)
```

Comment:
i think we should use `\A` instead of `^` since we may accidentally match middle of the string after a newline.

Reply:
Hi,

It has been fixed by this [commit](https://github.com/vmware-tanzu/carvel-vendir/pull/65/commits/de33a3d91927326a3f4bbf80ec2768f011cc9006).